app/views/jobs/index.html.erb

```diff
@@ -2,4 +2,13 @@
 
 <h1>Listing Jobs</h1>
 
+<%= form_tag("/jobs", method: "get") do %>
```

Comment:
The Rails way is to always stick to url helper methods in views. In this case, `jobs_path`.